dataStorage/dataStorage.py

```diff
@@ -32,9 +41,9 @@ def _store_json(location, json):
     """
     try:
         pickle.dump(json, open(location, "wb"), pickle.HIGHEST_PROTOCOL)
```

Comment:
Pickle is not a valid long-term solution for long-term storage, plus reliance on the structure of the classes could cause development problems if the stored files are not cleared when potentially breaking changes are made to class structures.